### DISCUSSION TITLE
Having big issue training with Illuminati model

### DISCUSSION THREAD

**Discussion by WalidAliAlfaid** [geocine/sd-easy-mode#9] [Announcements]
the model base is 2.1 768 it loads fine but the generation come out just noise

**Comment by geocine** [comment]
Sorry @WalidAliAlfaid I have not been using 2.1 and have not tested this with it. I have not get any good result on an 2.x model personally so I won't be able to help with this.

**Comment by WalidAliAlfaid** [comment]
thanks, one last thing can you add upload the trained model to HuggingFace option please, it would help alot!

**Reply by geocine** [reply]
I transferred to discussion. I might or might not work on this depending on my time. Thanks for suggestion
